sdk/clientcore/ci.yml

```diff
@@ -30,6 +30,10 @@ parameters:
     displayName: 'http-okhttp3'
     type: boolean
     default: false
+  - name: release_clientcoreannotationprocessor
```

Comment:
nit: Do we need clientcore prefix?

Reply:
I kept it as it was in the group id and that is the convention we followed for other packages.